client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -19,23 +19,27 @@ module.exports = React.createClass( {
 		post: React.PropTypes.object.isRequired,
 		includeNonDraftStatuses: React.PropTypes.bool,
 		link: React.PropTypes.string,
-		target: React.PropTypes.string
+		target: React.PropTypes.string,
+		iconSize: React.PropTypes.oneOf( [ 18, 24 ] ),
 	},
 
 	getDefaultProps: function() {
 		return {
 			includeNonDraftStatuses: false,
 			link: null,
-			target: null
+			target: null,
+			iconSize: 18
 		};
 	},
 
 	getRelativeTimeText: function() {
-		const status = this.props.post.status;
+		var status = this.props.post.status,
```

Comment:
Let's avoid that [var](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations):
```javascript
const { iconSize, post } = this.props;
const status = post.status;
let time, timeReference;
```